packages/react-components/react-field/src/components/Field/Field.types.ts

```diff
@@ -22,22 +24,15 @@ export type FieldSlots = {
   label?: Slot<typeof Label>;
 
   /**
-   * A message about the validation state. The appearance of the `validationMessage` depends on `validationState`.
-   */
-  validationMessage?: Slot<'div'>;
-
-  /**
-   * The icon associated with the `validationMessage`. If the `validationState` prop is set, this will default to an
-   * icon corresponding to that state.
-   *
-   * This will only be displayed if `validationMessage` is set.
+   * A message about the validation state. By default, this is an error message, but it can be a success, warning,
+   * or neutral message by setting `validationState`.
    */
-  validationMessageIcon?: Slot<'span'>;
+  validationMessage?: Slot<typeof FieldMessage>;
 
   /**
    * Additional hint text below the field.
    */
-  hint?: Slot<'div'>;
+  hint?: Slot<typeof FieldMessage>;
```

Comment:
Is there a benefit to making `hint` a `FieldMessage`? It seems maybe a bit weird to have `validationState` on a hint